rdmo/management/assets/js/components/common/Links.js

```diff
@@ -103,6 +103,26 @@ LockedLink.propTypes = {
   disabled: PropTypes.bool
 }
 
+const ToggleCurrentSiteLink = ({ has_current_site, locked, onClick, disabled }) => {
+  const className = classNames({
+    'element-btn-link fa': true,
+    'fa-plus-square-o': !has_current_site,
```

Comment:
The icons are not super, but I don't have a better suggestion. Maybe the transition to material symbols will solve this at one point.

Reply:
ok maybe like this? 
![image](https://github.com/rdmorganiser/rdmo/assets/1799675/5970cc3f-547c-4e0e-b846-f91fee747cab)

I also think it is better when the toogle site link is not the first one.

Reply:
`fa-plus-square` and `fa-minus-square`